internal/oci/runtime_vm.go

```diff
@@ -944,7 +944,8 @@ func (r *runtimeVM) AttachContainer(ctx context.Context, c *Container, inputStre
 		},
 	}
 
-	return cInfo.cio.Attach(opts)
+	cInfo.cio.Attach(opts)
+	return nil
```

Comment:
just a note for reviewers: we previously generated an error only if utils.GenerateID() failed (just the generation of a unique ID).
Also if we encountered an error while performing the Attach(), we logged it but returned nil.
So, looks fine to me always returning nil (as containerd's Attach() doesn't return any error).